drivers/remote/driver_test.go

```diff
@@ -527,6 +558,10 @@ func (r *rollbackEndpoint) AddressIPv6() *net.IPNet {
 	return nil
 }
 
+func (r *rollbackEndpoint) IPAliases() []*net.IPNet {
+	return nil
```

Comment:
This is just a dummy method, but it is actually implemented and tested in the other test case.